Subscriber/ExceptionSubscriber.php

```diff
@@ -16,39 +18,51 @@
  */
 class ExceptionSubscriber implements EventSubscriberInterface
 {
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
     /**
      * @var LoggerInterface
      */
     private $logger;
 
-    public function __construct(LoggerInterface $logger)
+    public function __construct(KernelInterface $kernel, LoggerInterface $logger)
     {
+        $this->kernel = $kernel;
         $this->logger = $logger;
     }
 
-    /**
-     * @return array
-     */
     public static function getSubscribedEvents(): array
     {
         return [
             KernelEvents::EXCEPTION => 'onKernelException'
         ];
     }
 
-    /**
-     * @param GetResponseForExceptionEvent $event
-     */
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
         $exception = $event->getException();
 
         $this->logger->log('error', $exception->getMessage());
 
         $response = new Response();
-        $response->setContent(json_encode(['errors' => $this->getErrorBody($exception)]));
         $response->setStatusCode(Response::HTTP_INTERNAL_SERVER_ERROR);
 
+        if ($this->kernel->isDebug()
+            || (
+                ($exception instanceof HttpExceptionInterface || $exception instanceof HttpException)
```

Comment:
we should also take into account multi part exceptions

Reply:
`Wizards\RestBundle\Exception\MultiPartHttpException` extends `WizardsRest\Exception\HttpException`.